modules/sdk-core/src/bitgo/lightning/lightning.ts

```diff
@@ -49,7 +47,7 @@ export class Lightning implements ILightning {
     throw new Error('method not implemented');
   }
 
-  public async getInvoices(params?: GetInvoicesParams): Promise<GetInvoicesResponse> {
+  public async getInvoices(): Promise<GetInvoicesResponse> {
```

Comment:
Let's remove until we're ready to implement this and we know what the params should be.